automatic/seafile-client/tools/chocolateyInstall.ps1

```diff
@@ -1,15 +1,24 @@
-﻿$packageName = '{{PackageName}}'
-$fileType = 'msi'
-$silentArgs = '/passive'
-$url = '{{DownloadUrl}}'
-$version = '{{PackageVersion}}'
+﻿$ErrorActionPreference = 'Stop'
+
+$packageArgs = @{
+  packageName = 'seafile-client'
+  fileType    = 'msi'
+  softwareName = 'Seafile 6.0.0'
+
+  checksum     = '64f0e7192acbbbad4e6ac34d5b531ff27697172b9109f2c00cf61ebae923a642'
+  checksumType = 'sha256'
+  url          = 'https://bintray.com/artifact/download/seafile-org/seafile/seafile-6.0.0-en.msi'
+
+  silentArgs   = '/passive'
+  validExitCodes = @(0)
+}
 
 $app = Get-WmiObject -Class Win32_Product | Where-Object {
```

Comment:
It probably would be better to use a registry key, but I just reused this as it was the check already used originally.
The check is necessary because of #70, sure could add 1603 as a valid exit code.
But since 1603 is considered a fatal error, that wouldn't be correct.